spec/services/invoices/payments/stripe_service_spec.rb

```diff
@@ -138,6 +146,8 @@
       let(:stripe_customer) { create(:stripe_customer, customer:) }
 
       before do
+        allow(stripe_service).to receive(:update_payment_method_id).and_return(nil)
```

Comment:
RSpec/SubjectStub: Do not stub methods of the object under test. (https://rspec.rubystyle.guide/#dont-stub-subject, https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/SubjectStub)